internal/db/migrate.go

```diff
@@ -0,0 +1,26 @@
+package db
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	migrate "github.com/rubenv/sql-migrate"
+	"github.com/stellar/wallet-backend/internal/db/migrations"
+)
+
+func Migrate(databaseURL string, direction migrate.MigrationDirection, count int) (int, error) {
```

Comment:
Can we add the context here instead of creating a new one?

Reply:
Good call, added here 👍 : [2e5e3d0](https://github.com/stellar/wallet-backend/pull/18/commits/2e5e3d01baf15920f91f889987766224a92b723e)